### PR TITLE
Project page for the "Epigraphing the 19th C."

### PR DESCRIPTION
**Pull request by aaronplasek**
Here's one of the projects I've been working on. Happy to hear any comments / suggestions / criticisms! And, as always, collaborators are very welcome!

### CODE REVIEW

**Comment by JonathanReeve** [comment]
Looks good to me! Will defer to others to merge.

File: _posts/events/2015-09-17-johanna-drucker.md [hunk 1]
```diff
@@ -18,6 +18,6 @@ Johanna Drucker, Breslauer Professor of Bibliographical Studies in the Departmen
 
 Professor Drucker outlines her concerns:
```
Comment (denten) [inline _posts/events/2015-09-17-johanna-drucker.md]: Separate the spelling and the new page into separate pull request.

File: _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+layout: project
+title: "Epigraphing the 19th Century"
+tags:
+- Aaron Plasek
+category: theory-method
+type: experiment / archive
```
Comment (denten) [inline _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md]: we only support one label for now

File: _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+published: true
+snippet: 123
+images:
+
+---
+
+#PROVOCATION
```
Comment (denten) [inline _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md]: Provocation format is for encounters.

File: _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+Frequently ignored and occasionally made up, the epigraph is a textual genre defined both by its physical placement on the page and by the absence of the textual object being signposted. An epigraph attribution situates the text it prefaces within a larger constellation of texts and authors, and in this manner has an indexical function rather similar to scanning the spines of books on a shelf, flipping through a card catalog, or examining a record in a digital relational database. While in no way accepting that the affordances offered through the iterative construction of citation networks is ever a substitution for a critical method, a comparative approach to the different kinds of citations practices made visible by different networks of attribution provide an opportunity to reconsider how shared concepts that constitute a (disciplinary) field are produced, transmitted, and inhibited through our media-centric notions of consensus.
```
Comment (denten) [inline _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md]: I like this for the opening better.

File: _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+Frequently ignored and occasionally made up, the epigraph is a textual genre defined both by its physical placement on the page and by the absence of the textual object being signposted. An epigraph attribution situates the text it prefaces within a larger constellation of texts and authors, and in this manner has an indexical function rather similar to scanning the spines of books on a shelf, flipping through a card catalog, or examining a record in a digital relational database. While in no way accepting that the affordances offered through the iterative construction of citation networks is ever a substitution for a critical method, a comparative approach to the different kinds of citations practices made visible by different networks of attribution provide an opportunity to reconsider how shared concepts that constitute a (disciplinary) field are produced, transmitted, and inhibited through our media-centric notions of consensus.   
+
+
+#WHAT WE'VE DONE
```
Comment (denten) [inline _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md]: Watch the royal plural. Use double hashes for subheadings.

File: _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+
+Collecting nearly 3,000 XML-encoded 19th-century US novels from the Early American Fiction archive and [Wright American Fiction](https://github.com/iulibdcs/tei_text) archive, Aaron (in [collaboration with others](https://github.com/aaronplasek/19th_C._novel_scraper/graphs/contributors)) wrote a [scraper in python](https://github.com/aaronplasek/19th_C._novel_scraper) to collect the following epigraph and bibliographic information: 
+(1) novel author,
```
Comment (denten) [inline _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md]: Not sure if this list format is correct markdown. Did you build locally? (jekyll serve -u)

File: _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+(2) novel title,
+(3) novel publication date,
+(4) novel publication location,
+(5) epigraph text,
+(6) epigraph attribution,
+(7) author birth and death years, and 
+(8) file provenance. 
+This information is useful both for exploring epigraph-usage and for examining broader questions about the circulation and printing of novels in the 19th century, and could be quite easily adapted to digital techniques like mapping. **So far we've collected 7,689 epigraphs (as of 11 Oct 2015).**
```
Comment (denten) [inline _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md]: Move the updates to the top under ## updates heading (I'll talk about this today)

File: _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+
+
+
+#EXPERIMENTAL CHALLENGES / OPPORTUNITIES FOR COLLABORATION
```
Comment (denten) [inline _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md]: overuse of "experimental" (be thrifty with the good words)

File: _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md [hunk 9]
```diff
@@ -0,0 +1,48 @@
...
+
+#SELECTED TALKS / PUBLICATIONS / PRESS
```
Comment (denten) [inline _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md]: this is great

File: _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md [hunk 10]
```diff
@@ -0,0 +1,48 @@
...
... for example, at least 8 different spellings of Shakespeare. To make matters worse, epigraph attributions formats are similarly unpredictable. Shakespeare's "Hamlet," for instance, might receive the following epigraph attributions: (1) Hamlet; (2) Shakespeare; (3) Shakespeare's Hamlet; (4) Hamlet, Shakespeare; (5) William Shakespeare; (6) William Shakespeare's Hamlet; and so forth. Merely separating the titles of the works (when present) from the author names (when present) is itself a very complex problem with no uniformly agreed upon solution. Such ambiguities must be resolved if one wishes to produce a definitive graphical model of epigraph attributions for our particular corpus. One potential solution is to crowd-source this work, as must be done with epigraph mistagging. We wish to open this corpus to the public for additional editing and future research. (Anyone interested in collaborating on this should send an email to [Aaron](http://aaronplasek.com).) However, another potential solution to this challenge is to perform this name disambiguation algorithmically. Such a project might make a very interesting undergraduate thesis project in computer science, math, or statistics.
```
Comment (denten) [inline _posts/projects/2015-10-4-epigraphing-19th-C.-novels.md]: I would shorten and tighten this up in general. You may want to include a Call for Participation, as it seems like you are looking for collaborators.

File: _posts/events/2015-09-17-johanna-drucker.md [hunk 11]
```diff
@@ -18,6 +18,6 @@ Johanna Drucker, Breslauer Professor of Bibliographical Studies in the Departmen
...
+> I am interested in modeling temporal relations without a standard Cartesian grid as the ground, finding a way to realize the unfulfilled project I began at \[The University of\] Virginia but couldn't realize because we did not have the technical ability. The manuscripts of [Eusebius's chronologies,](http://www.historyofinformation.com/expanded.php?id=3804) for instance, float their relative time scales so they don't have to be reconciled within a single metric standard. Other aspects of humanistic representation of temporal experience are variable granularity and non-continuous temporality. The issues of non-linear, multi-directional, non-homogenous, and variably scaled temporalities are ones I have thought about for quite awhile.
```
Comment (denten) [inline _posts/events/2015-09-17-johanna-drucker.md]: Heya Aaron, we are gong to use your PR for our code review model. Well done!

File: impact.md [hunk 12]
```diff
@@ -49,10 +49,10 @@ receive $70,000 from the Brown Institute for Media Innovation Magic Grant.
 
 ## <a name="placements"></a>Placements
 
-> *Aaron Plasek* admitted Columbia History Ph.D. program (Spring 2015).
+> *Aaron Plasek* admitted to the Columbia History Ph.D. program (Spring 2015).
```
Comment (denten) [inline impact.md]: good catch---separate into a separate PR pls (they should be grouped semantically, otherwise it becomes difficult to revert)
